website/docs/r/emr_cluster.html.markdown

```diff
@@ -668,12 +669,16 @@ EOF
 * `termination_protection` - (Optional) Switch on/off termination protection (default is `false`, except when using multiple master nodes). Before attempting to destroy the resource when termination protection is enabled, this configuration must be applied with its value set to `false`.
 * `visible_to_all_users` - (Optional) Whether the job flow is visible to all IAM users of the AWS account associated with the job flow. Default value is `true`.
 
-### bootstrap_action
+### auto_termination_policy
```

Comment:
It seems the name of this section and the one below (`bootstrap_action`) were accidentally switched in this PR.